Formula/odpi.rb

```diff
@@ -0,0 +1,22 @@
+class Odpi < Formula
+  desc "Oracle Database Programming Interface for Drivers and Applications"
+  homepage "https://oracle.github.io/odpi/"
+  url "https://github.com/oracle/odpi.git",
```

Comment:
We strongly prefer to use github tarballs (`https://github.com/oracle/odpi/archive/v2.1.0.tar.gz`) to download the source (allows caching and checksum control).

Reply:
Sure, fixed.